src/engine-actions/prepare-swap.js

```diff
@@ -23,23 +14,32 @@ const MEMO_PREFIX = 'sparkswap-swap-pivot:'
  * Prepares for a swap in which this node is the counterparty to the intiating node
  *
  * @param {string} orderId - Order for which the swap is being executed
- * @param {string} swapHash        - swap hash that will be associated with the swap
- * @param {string} value   - Int64 string of the value of inbound currency
- * @returns {string} swapHash      Hash that will be translated
+ * @param {string} swapHash - swap hash that will be associated with the swap
+ * @param {string} value - Int64 string of the value of inbound currency
+ * @param {number} expiry - payment request expiry time in seconds
+ * @param {number} cltvExpiry - delta to use for the time-lock of the CLTV
+ *                              extended to the final hop (in blocks)
+ * @returns {string} paymentRequest - lightning invoice for a payment
  */
-async function prepareSwap (orderId, swapHash, value) {
+async function prepareSwap (orderId, swapHash, value, expiry, cltvExpiry) {
```

Comment:
In https://github.com/sparkswap/broker/pull/547 we're no longer passing `orderId` as a parameter.

Reply:
So we don't need it in the memo anymore?

Reply:
It was always a nice to have for debugging, but I think the hash is good enough.